WalletWasabi.Fluent/Controls/CurrencyEntryBox.axaml.cs

```diff
@@ -273,7 +273,7 @@ private async void DoPasteCheckAsync(KeyEventArgs e)
 
 		public async void ModifiedPaste()
 		{
-			var text = await AvaloniaLocator.Current.GetService<IClipboard>().GetTextAsync();
+			var text = (await AvaloniaLocator.Current.GetService<IClipboard>().GetTextAsync()).Replace("\r\n", "");
```

Comment:
kinda hackey haha; Did we try to use `AcceptsReturn="False` on this one?

Reply:
I tried but didn't work... neither on the control nor as style on the TextPresenter.

Reply:
wont work for linux line endings... need to also replace `\n`

Reply:
done